roles/docker/tasks/main.yml

```diff
@@ -36,6 +36,15 @@
   tags:
     - prebake-for-dev
 
+- stat: path=/var/docker_images
+  register: docker_images
```

Comment:
the only function this tag offers is to pre-bake things in the packer image (to lower ansible provision times in vagrant environments) while not special casing them and using same ansible. So it's function is pretty limited. If there is a better way to accomplish this I would be happy to explore that option.